apis/Google.Cloud.Dataproc.V1/docs/history.md

```diff
@@ -0,0 +1,10 @@
+# Version history
+
+# Version 1.1.0, released 2019-12-09
+
+- [Commit 4e38a24](https://github.com/googleapis/google-cloud-dotnet/commit/4e38a24): Autoscaling and security configuration
+- [Commit 50658e2](https://github.com/googleapis/google-cloud-dotnet/commit/50658e2): Regenerate all APIs with resource name format methods
```

Comment:
nit: Maybe replace "Regenarate all APIs with" with "Add"

Reply:
Done. (That was just the commit message. Worth putting it in more context though.)